internal/integration_test/spectest/v2/spec_test.go

```diff
@@ -24,6 +24,13 @@ func TestCompiler(t *testing.T) {
 	spectest.Run(t, testcases, context.Background(), wazero.NewRuntimeConfigCompiler().WithCoreFeatures(enabledFeatures))
 }
 
+func TestCompilerNoDebug(t *testing.T) {
+	if !platform.CompilerSupported() {
+		t.Skip()
+	}
+	spectest.Run(t, testcases, context.Background(), wazero.NewRuntimeConfigCompiler().WithCoreFeatures(enabledFeatures).WithDebugInfoEnabled(false))
+}
+
```

Comment:
I guess spectest doesn't have debuginfo (dwarf) since it's coming from wat, not a real language, so I think we do not need to add this tests

```suggestion
```

Reply:
You're right, this is not the way.

I found these useful while developing the feature and wanted them to cover both cases.

Will remove.